pkg/controller/gittrack/handler_test.go

```diff
@@ -160,6 +159,18 @@ var _ = Describe("Handler Suite", func() {
 			})
 		}
 
+		var AssertIgnoresWrongNamespaceChild = func(r *handlerResult) {
+			It("does not create a GitTrackObject for the child", func() {
+				m.Get(gto, consistentlyTimeout).ShouldNot(Succeed())
+			})
+
+			PIt("ignores the child resource", func() {
```

Comment:
This test is pending as the error message has been made more verbose as part of the move to introduce `ClusterGitTrack`s